src/locales/uk.js

```diff
@@ -151,11 +140,11 @@ const translations = {
             'Ви вибрали забагато файлів. Максимальна кількість %max%.',
           tooFewFiles: 'Ви вибрали %files%. Мінімальна кількість %min%.',
           clear: 'Видалити все',
-          done: 'Додати'
-        },
```

Comment:
Seems like you lost bracket

Reply:
> Seems like you lost bracket

Double-check. Everything is ok with bracket.

Reply:
I understand we have wrong structure for UK translation and you fix it ❤️
Big thanks for contribution!

Reply:
> I understand we have the wrong structure for UK translation and you fix it ❤️
> Big thanks for the contribution!

Yes, it was. Welcome, guys!